discord/permissions.py

```diff
@@ -495,10 +495,7 @@ def _set(self, key, value):
         self._values[key] = value
 
     def pair(self):
-        """Returns the (allow, deny) pair from this overwrite.
-
-        The value of these pairs is :class:`Permissions`.
-        """
+        """Tuple[:class:`Permissions`]: Returns the (allow, deny) pair from this overwrite."""
```

Comment:
```suggestion
        """Tuple[:class:`Permissions`, :class:`Permissions`]: Returns the (allow, deny) pair from this overwrite."""
```

It's a two element tuple.